### PR TITLE
Grammatical corrections

### PR DESCRIPTION
**Pull request by Leatryx**
Russian localization punctuation and grammatical corrections

### CODE REVIEW

File: translations/russian.json [hunk 1]
```diff
@@ -19,76 +19,73 @@
...
+    "SOLANUM: .. Is that Memory Statue active? I wish Ramie could see this. I want to check on its connection with <![CDATA[<color=orange>Ash Twin Project</color>]]>.": "СОЛАНУМ: ...Это что, статуя памяти? Активированная? Рами был бы вне себя от радости. Мне нужно проверить её связь с <![CDATA[<color=orange>проектом Час Пепла</color>]]>.",
 
     // Recorder TH
-    "SOLANUM: It's you, friend. I feel like my mind is clear now, but I can't explain this, even for myself.": "СОЛАНУМ: А, это ты, друг мой. Мне уже легче, мой разум кристалльно чист, но это странное ощущение. Я не могу объяснить это чувство, даже сама себе.",
-    "SOLANUM: I'm here, right? But how I am here? I thought about <![CDATA[<color=lightblue>Timber Hearth</color>]]> and now I am here.": "СОЛАНУМ: Я ведь правда здесь? Я просто подумала тогда о <![CDATA[<color=lightblue>Камельке</color>]]> и очутилась тут.",
```
Comment (MegaPiggy) [inline translations/russian.json:74]: Why did you remove 3 translations?

**Review by MegaPiggy — CHANGES_REQUESTED** [review]

File: translations/russian.json (hunk 1 above)
Comment (Leatryx) [reply translations/russian.json:74]: I guess by accident.
